README.md

```diff
@@ -1,4 +1,8 @@
-# ETA Notifications Node. Powered by Twilio - ASP.NET MVC
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
+ # ETA Notifications Node. Powered by Twilio - ASP.NET MVC
 
 [![Build status](https://ci.appveyor.com/api/projects/status/yqxays3pbnjsau25?svg=true)](https://ci.appveyor.com/project/TwilioDevEd/eta-notifications-csharp)
 
```

Comment:
Can you add a link to the tutorial after the description? 

https://github.com/twilio/docs-guidelines/tree/master/templates/README/node/express
You can find there a rough example of what the Readme should look like.